pkg/kubelet/qos/policy.go

```diff
@@ -37,18 +39,22 @@ const (
 // multiplied by 10 (barring exceptional cases) + a configurable quantity which is between -1000
 // and 1000. Containers with higher OOM scores are killed if the system runs out of memory.
 // See https://lwn.net/Articles/391222/ for more information.
-func GetContainerOOMScoreAdjust(pod *v1.Pod, container *v1.Container, memoryCapacity int64) int {
+func GetContainerOOMScoreAdjust(pod *v1.Pod, container *v1.Container, memoryCapacity int64) (int, error) {
+	if memoryCapacity == 0 {
```

Comment:
should it be `>0` to make code even safer?

Reply:
hm probably - (well `<= 0`) - i'll update it in a bit

Reply:
> hm probably - (well <= 0) - i'll update it in a bit

@endocrimes can do it? Please, don't forget to update the correspondent test case(s).

Reply:
@endocrimes gentle ping

Reply:
oh yep - i've been super busy - i'll get to it tomorrow

Reply:
Updated 👍